### ISSUE TITLE
Traefik configuration to get Let's Encrypt certificate

### ISSUE THREAD

**Issue by nqb** [lefilament/ansible_role_docker_server#6]
Hello,

After I used this role + ansible_role_odoo_server, I'm unable to get an SSL certificate from Let's Encrypt using Traefik.

I always got this error message when using `docker logs inverseproxy`:

```
time="2021-03-12T00:02:59+01:00" level=error msg="Unable to obtain ACME certificate for domains \"www.example.fr\": unable to generate a certificate for the domains [www.example.fr]: error: one or more domains had a problem:\n[www.example.fr] acme: error: 400 :: urn:ietf:params:acme:error:tls :: Fetching https://www.example.fr/.well-known/acme-challenge/1QKSyEHPggn17gKAUNApVUvokiZoiF6xxCzfTVgQ5qo: remote error: tls: internal error\n" providerName=le.acme routerName=websecure-odoo@docker rule="Host(`www.example.fr`)"
time="2021-03-12T00:03:08+01:00" level=error msg="Unable to obtain ACME certificate for domains \"www.example.fr\": unable to generate a certificate for the domains [www.example.fr]: error: one or more domains had a problem:\n[www.example.fr] acme: error: 400 :: urn:ietf:params:acme:error:tls :: Fetching https://www.example.fr/.well-known/acme-challenge/v917UVWCjVbpJaOc720uoSqA0aYrUzN7dym1s61ouyg: remote error: tls: internal error\n" routerName=websecure-odoo-restrict@docker rule="Host(`www.example.fr`) && Path(`/website/info`, `/web/database/{p:manager|create|duplicate|drop|backup|restore|change_password}`)" providerName=le.acme
```

It seems to be identical to https://github.com/traefik/traefik/issues/7825 which is fixed in Traefik v2.4.2 (see https://github.com/traefik/traefik/releases/tag/v2.4.2) but using 2.4.7 image, I still have an issue.

If I used `[certificatesResolvers.le.acme.tlsChallenge]`, Traefik got a cert from Let's Encrypt without issue.

**Comment by remi-filament** [comment]
I have observed the same issue with Traefik 2.4
I realized that some variable names have changed and also that TLS challenge worked better.
I implemented these in 0f726da7fc4ecbbc0a41596edbed557d551d4e18
I guess this should solve that issue, feel free to reopen it in case this is not the case

**Comment by nqb** [comment]
Thanks!
